dali/pipeline/executor/executor.cc

```diff
@@ -75,6 +75,7 @@ void Executor::RunCPU() {
   lock.unlock();
 
   // Run the support ops
+  DeviceGuard g(device_id_);
```

Comment:
Please put it above comment, now it is misleading.

Reply:
Fixed.